### PR TITLE
WSTEAMA-1152: Delete idx page code

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAMA-1152

Overall changes
======
Deletes all references to IDX pages

Code changes
======
Delete alllll the code

Testing
======
All automated tests passing

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/legacy/containers/StoryPromo/helpers/fixtureData.js [hunk 1]
```diff
@@ -269,7 +265,7 @@ export const mapWithMediaError = {
   },
   section: {
     name: 'کور پاڼه',
-    subType: INDEX_PAGE,
+    subType: 'IDX',
     type: 'simple',
     uri: '/pashto/front_page',
   },
```
Comment (amoore108) [inline src/app/legacy/containers/StoryPromo/helpers/fixtureData.js:271]: Just curious if these should be the `FRONT_PAGE` type? Not too familiar with these routes!

**Review by amoore108 — APPROVED** [review]
Very nice! 🗑️

File: src/app/legacy/containers/StoryPromo/helpers/fixtureData.js (hunk 1 above)
Comment (karinathomasbbc) [reply src/app/legacy/containers/StoryPromo/helpers/fixtureData.js:271]: In this particular instance, `/pashto/front_page` would have been returned as a subType of `IDX` from CPS, as front pages are just a type of `IDX` page. This fixture was taken from https://ares-api.api.bbci.co.uk/api/asset/52521615, where the `subType` is actually set to `index` but I don't think the `subType` is used at all in Simorgh, so this doesn't really matter. Plus this is just fixture data, used for tests 🤷‍♀️ 

Note also, that I have not removed any references in the data folder which contain `subType: 'IDX'` as I didn't want to mess with it!

Comment (amoore108) [reply src/app/legacy/containers/StoryPromo/helpers/fixtureData.js:271]: No probs, thanks for clarifying 👍

**Review by louisearchibald — APPROVED** [review]

**Pull request merged**